src/Auth0.ManagementApi/Models/PasswordChangeTicketRequest.cs

```diff
@@ -51,5 +51,14 @@ public class PasswordChangeTicketRequest
         /// </summary>
         [JsonProperty("includeEmailInRedirect")]
         public bool? IncludeEmailInRedirect { get; set; }
+
+        /// <summary>
+        /// ID of the client.
+        /// If provided for tenants using New Universal Login experience, 
```

Comment:
```suggestion
        /// If provided for tenants using the New Universal Login experience, 
```

Reply:
Thanks, copied it from the API v2 docs, so did not pay attention to the content. Will change it.